handling-null-values-in-avro-and-protobuf/kafka/build.gradle

```
@@ -0,0 +1,146 @@
+buildscript {
+    repositories {
+        mavenCentral()
+        maven {
+            url = uri("https://packages.confluent.io/maven/")
+        }
+        maven {
+            url = uri("https://plugins.gradle.org/m2/")
+        }
+        maven {
+            url = uri("https://jitpack.io")
+        }
+    }
+}
+
+plugins {
+    id 'java'
+    id 'idea'
+    id 'eclipse'
+    id "com.google.protobuf" version "0.9.4"
+    id "com.github.imflog.kafka-schema-registry-gradle-plugin" version "1.13.0"
+    id "com.github.davidmc24.gradle.plugin.avro" version "1.3.0"
+}
+
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
```

for consistency with the rest of the repo

```suggestion
java {
    sourceCompatibility = JavaVersion.VERSION_17
    targetCompatibility = JavaVersion.VERSION_17
}
```